python/dgl/data/chem/utils.py

```diff
@@ -0,0 +1,175 @@
+import dgl.backend as F
+import numpy as np
+import os
+import pickle
+
+from dgl import DGLGraph
+
+try:
+    from rdkit import Chem
+    from rdkit.Chem import rdmolfiles, rdmolops
+except ImportError:
+    raise ImportError("Please install rdkit")
+
+def one_hot_encoding(x, allowable_set):
+    """One-hot encoding.
+
+    Parameters
+    ----------
+    x : str, int or Chem.rdchem.HybridizationType
+    allowable_set : list
+        The elements of the allowable_set should be of the
+        same type as x.
+
+    Returns
+    -------
+    list
+        List of boolean values where only one value is True.
```

Comment:
It's possible that the whole vector is 0. I'd replace "only one" by "at most one".